### PR TITLE
Glassfish RI code-scanning fixes (excluding appserver/tests)

### PR DESCRIPTION
**Pull request by vinayvishal**
Generated new cacerts.jks files with self signed certificates and replaced existing ones with these ones.
Incorporated review comments.
Updated/removed images from the repository.

### CODE REVIEW

File: nucleus/osgi-platforms/osgi-cli-interactive/pom.xml [hunk 1]
```diff
@@ -62,13 +62,6 @@
         </developer>
     </developers>
 
-    <licenses>
-        <license>
-            <name>CDDL + GPL 1.1</name>
-            <url>http://glassfish.java.net/public/CDDL+GPL_1_1.html</url>
-        </license>
-    </licenses>
-
```
Comment (bshannon) [inline nucleus/osgi-platforms/osgi-cli-interactive/pom.xml]: Why is this license entry being removed?

**Review by bshannon — DISMISSED** [review]
Generally looks good to me.  One minor question below.

Please confirm the image file changes with Anissa.

**Comment by vinayvishal** [comment]
@glassfishrobot Run CI tests please

**Comment by vinayvishal** [comment]
@glassfishrobot Run CI tests please

File: nucleus/osgi-platforms/osgi-cli-interactive/pom.xml (hunk 1 above)
Comment (vinayvishal) [reply nucleus/osgi-platforms/osgi-cli-interactive/pom.xml]: This is based on one of your review comments.
> #28 - Yes, the license information in the pom.xml file will need to be updated.  Or just remove it and inherit it from the parent.

Comment (bshannon) [reply nucleus/osgi-platforms/osgi-cli-interactive/pom.xml]: Ah, ok, that's what I suspected.  Thanks!

**Comment by vinayvishal** [comment]
@glassfishrobot Run CI tests please

**Comment by yaminikb** [comment]
@glassfishrobot run ci tests please

**Review by yaminikb — APPROVED** [review]

**Comment by yaminikb** [comment]
@glassfishrobot run ci tests please

**Pull request merged**